### DISCUSSION TITLE
Way to tell if handle is currently selected?

### DISCUSSION THREAD

**Discussion by zachintosh** [xyflow/xyflow#3236] [Q&A] [answered]
By default, the user can click on a handle and then click on another handle to create an edge between them. Is there a way to know if a handle is currently "selected" and will be connected to another handle if clicked? I'd like to style the first handle in a way that lets the user know they are going to connect it to another handle if they click on another handle.

Love the library, by the way. Excellent work.

**Comment by bcakmakoglu** [comment, marked as answer]
There's a value that's stored called `connectionClickStartHandle` which you can check for the "selected" handle.

**Reply by zachintosh** [reply]
~@bcakmakoglu Where is `connectionClickStartHandle` stored?~ Forgot `useStore` is a thing. Thank you!

**Reply by moklick** [reply]
we also add a class "connecting" to a handle.
